contrib/junit.tpl

```diff
@@ -16,7 +16,7 @@
     </testsuite>
 
 {{- if .MisconfSummary }}
-    <testsuite tests="{{ add .MisconfSummary.Successes .MisconfSummary.Failures }}" failures="{{ .MisconfSummary.Failures }}" name="{{  .Target }}" errors="0" skipped="{{ .MisconfSummary.Exceptions }}" time="">
+    <testsuite tests="{{ add .MisconfSummary.Successes .MisconfSummary.Failures }}" failures="{{ .MisconfSummary.Failures }}" name="{{  .Target }}" errors="0" time="">
```

Comment:
can we add `skipped="0"` instead of `.MisconfSummary.Exceptions`?

Reply:
Essentially we still have the option to ignore misconfigurations. Wouldn't `skipped=“0”` be confusing?

Reply:
yes, you are right.
I was confused that we use `skipped="0"` for vulnerabilities.

Looks like we need to update our logic for `junit` template (in another PR of course)